mitodl/open-discussions
# moderator post notifications

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/open-discussions/issues/3165

#### What's this PR do?
Adds a checkbox to `http://localhost:8063/manage/c/edit/<channel>/basic/` to enable moderator notifications for posts
Sends notifications letting channel moderators know when there are posts in their channel if moderator notifications are enabled

#### How should this be manually tested?
Go to `http://localhost:8063/manage/c/edit/<channel>/basic/` to enable the feature
Create a new post in the channel
```
from notifications.tasks import *
send_unsent_email_notifications()
```
You should get a notification on the MAILGUN_RECIPIENT_OVERRIDE email

#### Screenshots (if appropriate)
<img width="1650" alt="Screen Shot 2020-11-16 at 11 14 39 AM" src="https://user-images.githubusercontent.com/1934992/99280380-41dbce00-27ff-11eb-9ae4-2c679f278423.png">
<img width="929" alt="Screen Shot 2020-11-16 at 11 20 22 AM" src="https://user-images.githubusercontent.com/1934992/99280404-4902dc00-27ff-11eb-8f43-4c40d50306a8.png">
